nixio/pycore/entity.py

```diff
@@ -67,31 +66,8 @@ def __str__(self):
     def __repr__(self):
         return self.__str__()
 
-util.create_h5props(NamedEntity, ("name", "type", "definition",
-                                  "id", "created_at", "updated_at"))
+util.create_h5props(NamedEntity,
```

Comment:
The signature of `create_h5props` with a set of names and a set of types is a bit ugly. Just use a dict instead:

```python
util.create_h5props(NamedEntity, {"name": str, "type": str, "definition": str})
```

Or you could implement `create_h5props` using **kwargs:

```python
util.create_h5props(NamedEntity, name: str, type: str, definition: str)
```